activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*    Add `Enumerable#maximum` and `Enumerable#minimum` to easily calculate the maximum or minimum from extracted
+     elements in an enumerable.
+
+     ```ruby
+     payments = [Payment.new(5), Payment.new(15), Payment.new(10)]
+
+     payments.minimum(:price) # => 5
+     payments.maximum(:price) # => 20
+     ```
+
```

Comment:
Let's cut the Action Pack changelog entry and add a passing mention here: "This also allows passing enumerables to `fresh_when` and `stale?` in Action Controller. See the PR https://github.com/rails/rails/pull/41404 for an example."

Reply:
Updated the PR description (with a polished example) and removed this entry